src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -17,6 +17,7 @@ public class EasyPostClient {
     public final BetaRateService betaRate;
     public final BillingService billing;
     public final CarrierAccountService carrierAccount;
+    public final CarrierMetadataService carrierMetadata;
```

Comment:
nit: Should put a deprecated warning on the `public final BetaCarrierMetadataService betaCarrierMetadata` element on line 15.